web-server/src/components/Service/SystemStatus.tsx

```diff
@@ -0,0 +1,167 @@
+import { Box, CircularProgress, Divider } from '@mui/material';
+import { FC, useEffect } from 'react';
+
+import { ServiceNames } from '@/constants/service';
+import { CardRoot } from '@/content/DoraMetrics/DoraCards/sharedComponents';
+import { serviceSlice, ServiceStatusState } from '@/slices/service';
+import { useDispatch, useSelector } from '@/store';
+
+import { FlexBox } from '../FlexBox';
+import { useOverlayPage } from '../OverlayPageContext';
+import { Line } from '../Text';
+
+export const SystemStatus: FC = () => {
+  const dispatch = useDispatch();
+  const loading = useSelector((state) => state.service.loading);
+  const services = useSelector(
+    (state: { service: { services: ServiceStatusState } }) =>
+      state.service.services
+  );
+
+  useEffect(() => {
+    const eventSource = new EventSource(`/api/stream`);
+    eventSource.onmessage = (event) => {
+      const data = JSON.parse(event.data);
+
+      if (data.type === 'status-update') {
```

Comment:
same, these string literals should be stored as const or Enums

Reply:
Sure